Library/Formula/justrelease.rb

```diff
@@ -0,0 +1,15 @@
+class Justrelease < Formula
+  desc "Install JustRelease Tool as CLI"
```

Comment:
It should be a description of Justrelease, not of the formula. You could use e.g. “CLI tool to release software libraries hosted on github.com”.